pyitau/main.py

```diff
@@ -87,6 +91,71 @@ def get_statements_from_month(self, month=1, year=2001):
         )
         return response.json()
 
+    def get_pix(self, days=90):
+        """Get and return the pix transaction for the last {days}."""
+
+        response = self._session.post(
+            ROUTER_URL, headers={"op": self._bigger_menu_page.pix_statements_op})
+        self.pix_page = PixPage(response.text)
+
+        data = {
+            "periodo": f"{days}",
+            "page": "1",
+            "filtro": "debito",
+            "ordenacao": "desc",
+        }
+
+        headers = {
+            "Op": self.pix_page.pix_statements_op,
+            "X-FLOW-ID": self._flow_id,
+            "X-CLIENT-ID": self._client_id,
+            "X-Requested-With": "XMLHttpRequest",
+            "Accept": "application/json, text/javascript, */*; q=0.01",
+            "Content-Type": "application/x-www-form-urlencoded; charset=UTF-8",
+        }
```

Comment:
**Suggestion:** Avoid using magic strings. Instead, define them as constants at the top of your file or class for better readability and maintainability.
```suggestion
        ACCEPT_HEADER_VALUE = "application/json, text/javascript, */*; q=0.01"
        CONTENT_TYPE_HEADER_VALUE = "application/x-www-form-urlencoded; charset=UTF-8"
        headers = {
            "Op": self.pix_page.pix_statements_op,
            "X-FLOW-ID": self._flow_id,
            "X-CLIENT-ID": self._client_id,
            "X-Requested-With": "XMLHttpRequest",
            "Accept": ACCEPT_HEADER_VALUE,
            "Content-Type": CONTENT_TYPE_HEADER_VALUE,
        }
```